module/components/Avatar/Readme.md

```diff
@@ -2,8 +2,8 @@ Avatar:
 
 ```js
 <div>
-    <Avatar color="info" src='styleguide/avatar_01.jpg' /> {' '}
-    <Avatar color="info" src='styleguide/avatar_02.jpg' />
+    <Avatar color="info" src='styleguide/avatar_01.jpg' alt="Jane Doe" /> {' '}
+    <Avatar color="info" src='styleguide/avatar_02.jpg' alt="John Doe" />
```

Comment:
Why both are "John Doe" ?

Reply:
both are not the same one is Jane and the other is John Doe